lib/resultuploaders/s3compatible/s3compatible.rb

```diff
@@ -0,0 +1,199 @@
+# frozen_string_literal: true
+
+require 'aws-sdk-s3'
+require './lib/auxiliary/json_helper'
```

Comment:
Please remove this as they are now autoloaded. See: https://github.com/HCK-CI/AutoHCK/pull/407

Reply:
ok